thoth/glyph/exceptions.py

```diff
@@ -0,0 +1,8 @@
+class RepositoryNotFoundException(Exception):
+    pass
+
+class ModelNotFoundException(Exception):
+    pass
+
+class NoMessageEnteredException(Exception):
+    pass
```

Comment:
Usually, it's a good idea to create a hierarchy of exceptions. That way a user of a library can catch library-wide exception stating any error occurred or specific exceptions for errors. See for example:

https://github.com/thoth-station/adviser/blob/6c722df4568e8d7b263c4dd4c1979bf931a55b75/thoth/adviser/exceptions.py#L26-L27

The exception acts like a base for all the exceptions in thoth-adviser library/module.

Reply:
Hmm...as all the 3 exceptions are completely distinct from each other : RepositoryNotFoundException, ModelNotFoundException, NoMessageEnteredException; They should all extend from a common thoth-glyph exception? right?

Reply:
Pushed changes. All these 3 exceptions now extend from ThothGlyphException